core/crypto/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dependencies]
 arrayref = "0.3"
-blake2 = "0.9.1"
+blake2 = { git = "https://github.com/near/near-blake2.git", branch = "f-standard-out", version = "0.9.1" }
```

Comment:
Eventually need to republish this to `near-blake2` and then change to `{ package = "near-blake2", version = "0.9.1" }`, as this crate needs to be published for the simulator.